demo.css

```diff
@@ -12,5 +12,61 @@ body {
     margin-bottom: 2rem;
 }
 .summary a {
-    color: #41403e;    
-}
\ No newline at end of file
+    color: #41403e;
+}
+
+/* paper topnav START */
+.topnav {
```

Comment:
why has the css been included here? You should create a new file in the source folder and add the new nav bar css in there

Reply:
ok, I will add new file. Thank you for feedback!